### PR TITLE
Added doesNotHaveAttribute() and elementAttributeNotExists() with tests

### PR DESCRIPTION
**Pull request by J7mbo**
Should be really simple - it's just a few changes with negations. If you don't like the exception stuff however we can refactor that.

### CODE REVIEW

File: src/Element/NodeElement.php [hunk 1]
```diff
@@ -118,6 +118,18 @@ public function hasAttribute($name)
     }
 
     /**
+     * Checks whether element does not have an attribute with specified name.
+     *
+     * @param string $name
+     *
+     * @return Boolean
+     */
+    public function doesNotHaveAttribute($name)
```
Comment (stof) [inline src/Element/NodeElement.php]: -1 for this method. Use ``!$element->hasAttribute($name)`` in the WebAssert class

File: tests/WebAssertTest.php [hunk 2]
```diff
@@ -982,6 +982,81 @@ public function testElementAttributeExists()
         );
     }
 
+    public function testElementAttributeNotExistsValid()
+    {
+        $page = $this->getMockBuilder('Behat\\Mink\\Element\\DocumentElement')
+                     ->disableOriginalConstructor()
+                     ->getMock()
```
Comment (stof) [inline tests/WebAssertTest.php]: wrong indentation

File: tests/WebAssertTest.php [hunk 3]
```diff
@@ -982,6 +982,81 @@ public function testElementAttributeExists()
...
+        ;
+
+        $element
+            ->expects($this->exactly(1))
+            ->method('doesNotHaveAttribute')
+            ->with('blah')
+            ->will($this->returnValue(true))
+        ;
+
+        call_user_func_array(array($this->assert, 'elementAttributeNotExists'), array('css', 'h2 > span', 'blah'));
```
Comment (stof) [inline tests/WebAssertTest.php]: no need to use ``call_user_func_array`` here

Comment (stof) [inline tests/WebAssertTest.php]: btw, it would be better to sue the existing ``assertCorrectAssertion`` and ``assertWrongAssertion`` methods instead

Comment (J7mbo) [inline tests/WebAssertTest.php]: I was thinking of adding an extra parameter for the above methods to take into account this. What do you think, any ideas on that?

**Comment by J7mbo** [comment]
I really can't fix the indentation here :-/

**Comment by aik099** [comment]
You again have sent PR from master branch. You should close this one and send one from different branch.

**Pull request closed without merging**